src/euphorie/client/model.py

```diff
@@ -686,6 +685,14 @@ class AccountChangeRequest(BaseObject):
     )
 
 
+Account.change_request = orm.relationship(
```

Comment:
Why are you declaring this (and some other attributes)  outside of the class?

Reply:
When referencing a column, the relation requires to be defined after the referenced class is defined.

Reply:
Or you can reference it by name (string).

Reply:
I know that trick of using strings for the table class.

I tried:
- `account_id`
- `SurveySession.account_id`
- `session.account_id`

sqlalchemy.exc.ArgumentError: Column expression expected for argument 'foreign_keys'; got '$THE_STRING_I_TRIED'.

I did not dig deeper in this 
Feel free to make another PR to fix that if you know how to do it.

BTW, the SQLAlchemy documentation also has a similar example: https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#adding-relationships-to-mapped-classes-after-declaration

It suggests:
```
Parent.children = relationship(Child, primaryjoin=Child.parent_id == Parent.id)
```
which is more or less the same thing I am doing...